mlflow/pipelines/pipeline.py

```diff
@@ -187,14 +190,129 @@ def _get_step_classes(self):
         pass
 
     @experimental
-    @abc.abstractmethod
     def _get_pipeline_dag_file(self) -> str:
         """
         Returns absolute path to the pipeline DAG representation HTML file.
-
-        Concrete pipeline class should implement this method.
         """
-        pass
+        import jinja2
+
+        j2_env = jinja2.Environment(loader=jinja2.FileSystemLoader(os.path.dirname(__file__)))
+        pipeline_dag_template = j2_env.get_template("resources/pipeline_dag_template.html").render(
+            {
```

Comment:
We'll eventually add a `"template": self._template` entry here to facilitate logic in the Jinja template that is conditioned on the template type.